docs/playbook.md

```diff
@@ -18,6 +18,26 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
 ### Mount successful but files not visible
 This is explained in the [semantics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#implicit-directories) page and can possibly be fixed by passing the --implicit-dirs=true flag to the gcsfuse command.
 
+### Mounting Bucket doesn't get Succeeded
```

Comment:
Do you want to keep the title as mount failed with fusermount3 exit status 1 to keep in sync with other titles here.

Reply:
fixed